src/js/containers/ActiveZoneSelector/ActiveZoneSelector.js

```diff
@@ -40,16 +40,29 @@ class ActiveZoneSelector extends Component {
     }
 
     render() {
-        let { activeZone, intl, zoneList } = this.props;
+        let { activeZone, zoneList } = this.props;
         let zones = _.values(zoneList).map(zone => {
             return { value: zone.name, label: zone.name };
         });
 
+        let activeZoneName = activeZone.name;
+        let customStyle = { margin: "9px auto 0" };
+        if (zones.length > 1) {
+            customStyle = { margin: "3px auto 0" };
+        }
+
+        if (zones.length < 2) {
+            return (
+                <div style={ customStyle }>
```

Comment:
`customStyle` is confusing, I don't understand what styles are being applied and when.  I assume we're stying for single item vs list right?  If so lets do something like this:
```
let singleItemStyles = {...};
let listStyle = {...};

...
<div style={zones.length <= 1 ? singleItemStyle : listStyle}

```